configs/training/flowbot_artflownet.yaml

```diff
@@ -3,5 +3,6 @@ batch_size: 64
 epochs: 100
 n_proc: 32
 check_val_every_n_epoch: 1
-mask_channel:
+trajectory_len: 5
```

Comment:
shouldn't this be trajectory_len: 1?

Reply:
Oh yes it should be. But it actually didn't use this param I think